packages/slice-machine/package.json

```diff
@@ -38,8 +38,7 @@
   "dependencies": {
     "@babel/register": "^7.11.5",
     "@reach/menu-button": "^0.10.5",
-    "@slicemachine/core": "^1.0.0",
```

Comment:
weird O.O

Reply:
indeed looool, I just did a make clean and make lol